lib/packages/crypto/src/generateMasterKeys.ts

```diff
@@ -0,0 +1,23 @@
+import { Keys } from "@burst/core";
+import * as CryptoJS from "crypto-js";
+import { ECKCDSA } from "./ec-kcdsa";
+import { Converter } from "./converter";
+
+/*
+* Generate the Master Public Key and Master Private Key for a new passphrase
+* EC-KCDSA sign key pair + agreement key.
+*/
+export const generateMasterKeys = (passPhrase: string): Promise<Keys> => {
+    return new Promise((resolve, reject) => {
```

Comment:
I still doubt that it should be async. IMHO It'd be better to keep this sync and "promisify" when needed...but I'll make some research to understand better eventual reasons for async. The main problem I see, is that all may become inconsistent...some sync, some async and then our lib suffers a good usability.

Reply:
Agreed, let me try to fix.